data_model/src/events/time.rs

```diff
@@ -326,10 +339,10 @@ mod tests {
             // ----|------[-----)----*----
             //     p     i1    i2
 
-            let schedule = Schedule::starting_at(Duration::from_secs(TIMESTAMP))
-                .with_period(Duration::from_secs(10));
-            let since = Duration::from_secs(TIMESTAMP + 35);
-            let length = Duration::from_secs(4);
+            let schedule = Schedule::starting_at(Duration::from_secs(TIMESTAMP - 5))
+                .with_period(Duration::from_secs(20));
+            let since = Duration::from_secs(TIMESTAMP);
+            let length = Duration::from_secs(5);
             let interval = TimeInterval { since, length };
             assert_eq!(count_matches_in_interval(&schedule, &interval), 0);
```

Comment:
Does this change guarantee anything new?
In each case before and after, the intersection of schedule and interval is empty, regardless of where `schedule.start` is

Reply:
I think this tests isn't testing anything new.
In each case two interval is located between two schedule points.
I'm going to revert it.